client/src/components/login/login.css

```diff
@@ -8,4 +8,8 @@
     font-size: 30px;
     text-align: center;
     font-weight: bold;
+}
+
+.login-button-container {
+    margin: 20px auto;
```

Comment:
Can you add a screenshot of how the Login page renders?

Reply:
Done.